src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSBwaAlignerSpark.java

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.hellbender.tools.spark.pathseq;
+
+import htsjdk.samtools.SAMFileHeader;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.broadcast.Broadcast;
+import org.broadinstitute.hellbender.utils.bwa.BwaMemIndexSingleton;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+/**
+ * Wrapper class for using the PathSeq Bwa aligner class in Spark. Encapsulates closing the index when done.
+ */
+public final class PSBwaAlignerSpark implements AutoCloseable {
+
+    private final JavaSparkContext ctx;
+    private final Broadcast<PSBwaArgumentCollection> bwaArgsBroadcast;
```

Comment:
This is too small to broadcast.  It's already Serializable, so you can pass it via lambda capture instead.